src/Autofac/Core/Resolving/ResolveOperationBase.cs

```diff
@@ -34,169 +34,102 @@
 namespace Autofac.Core.Resolving
 {
     /// <summary>
-    /// Defines the base properties and behaviour of a resolve operation.
+    ///     Defines the base properties and behaviour of a resolve operation.
     /// </summary>
     public abstract class ResolveOperationBase : IResolveOperation
     {
         private const int SuccessListInitialCapacity = 32;
-
         private bool _ended;
-        private readonly List<ResolveRequestContext> _successfulRequests = new List<ResolveRequestContext>(SuccessListInitialCapacity);
+
+        private readonly List<IResolveRequestContext> _successfulRequests =
+            new List<IResolveRequestContext>(SuccessListInitialCapacity);
+
         private int _nextCompleteSuccessfulRequestStartPos;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ResolveOperationBase"/> class.
+        ///     Initializes a new instance of the <see cref="ResolveOperationBase" /> class.
         /// </summary>
-        /// <param name="mostNestedLifetimeScope">The most nested scope in which to begin the operation. The operation
-        /// can move upward to less nested scopes as components with wider sharing scopes are activated.</param>
+        /// <param name="mostNestedLifetimeScope">
```

Comment:
Is something odd going on with the line breaks here? Might just be a github thing.